pkg/pillar/cmd/nim/nim.go

```diff
@@ -106,7 +107,9 @@ type nim struct {
 	assignableAdapters types.AssignableAdapters
 	enabledLastResort  bool
 	forceLastResort    bool
+	forceManualDPC     bool
 	lastResort         *types.DevicePortConfig
+	manualDPC          *types.DevicePortConfig
```

Comment:
It seems that `manualDPC` is unused?

Reply:
@milan-zededa this was a placeholder to implement merging of DPCs, not it is done on rust side but I do not like to have so many dependencies on eve types. I'll remove it for now